libs/accounts/recovery-phone/src/lib/sms.manager.ts

```diff
@@ -29,12 +29,47 @@ export class SmsManager {
   ) {}
 
   public async phoneNumberLookup(phoneNumber: string) {
-    const result = await this.client.lookups.v2
-      .phoneNumbers(phoneNumber)
-      .fetch({ fields: this.config.extraLookupFields.join(',') });
-    // Calling toJSON converts PhoneNumberInstance into a
-    // object that just holds state and can be serialized.
-    return result.toJSON();
+    try {
+      this.metrics.increment('sms.phoneNumberLookup.start');
+      const result = await this.client.lookups.v2
+        .phoneNumbers(phoneNumber)
+        .fetch({ fields: this.config.extraLookupFields.join(',') });
+
+      this.metrics.increment('sms.phoneNumberLookup.success');
+
+      /**
+       * See https://www.twilio.com/docs/lookup/v2-api/sim-swap for more details
+       * on the sim swap object returned by Twilio. We currently track only
+       * the last sim swap period.
+       */
+      const simSwapPeriod = result.simSwap?.lastSimSwap?.swappedPeriod;
+      if (simSwapPeriod) {
+        this.metrics.increment('sms.phoneNumberLookup.success.simSwap', {
+          period: simSwapPeriod,
+        });
+      }
+      const smsPumpingRisk = result?.smsPumpingRisk?.smsPumpingRiskScore;
+      if (smsPumpingRisk) {
+        this.metrics.histogram(
+          'sms.phoneNumberLookup.success.smsPumpingRisk',
+          smsPumpingRisk
+        );
```

Comment:
I don't think I understand this metric.

Reply:
I was hoping that we could see the sms pumping risk over time. Histogram seems right?

https://www.twilio.com/docs/glossary/what-is-sms-pumping-fraud

Reply:
I think histogram is a right StatsD metric type for it.  But I'm trying to think how I'd interpret something like average sms pumping risk over time.  Maybe just a lack of imagination on my part.

Reply:
A gauge might also be useful, see comment above. That should give us a scatter plot of points over time.

Reply:
I say we keep both?

Reply:
Sounds good to me.